File: q1.js
[...Array(100).keys()].forEach(item => {
    const isMultipleThree = Number.isInteger(item / 3);
    const isMultipleFive = Number.isInteger(item / 5);
    if (isMultipleThree && isMultipleFive) {
        console.log('Explorica Tours');
    }
    else if (isMultipleThree) {
        console.log('Explorica');
    } else if (isMultipleFive) { 
        console.log('Tours');
    } else {
        console.log(item);
    }
});

File: q1.cs
//Prints the numbers 1-100. For multiples of 3, prints 'cli_apps' instead of the number. For multiples of 5, prints 'C#' instead of the number.
//For multiples of both 3 and 5, prints 'cli_apps C#'.

using System;
using System.Linq;

public class Q1
{
    public void init()
    {
        int[] intArr = Enumerable.Range(0, 100).ToArray();
        foreach (int i in intArr)
        {
            bool isMultipleThree = i % 3 == 0;
            bool isMultipleFive = i % 5 == 0;
            if (isMultipleThree && isMultipleFive)
            {
                Console.WriteLine("cli_apps C#");
            }
            else
            {
                if (isMultipleThree)
                {
                    Console.WriteLine("cli_apps");
                }
                else
                {
                    if (isMultipleFive)
                    {
                        Console.WriteLine("C#");
                    }
                    else
                    {
                        Console.WriteLine(i);
                    }
                }
            }
        }
    }
}


